### PR TITLE
SDK-21: Some core connectors fail to compile because sdk-api classes cannot be found.

### PR DESCRIPTION
**Pull request by SebaElizalde**

### CODE REVIEW

**Review by dabreu — DISMISSED** [review]

File: modules/ROOT/pages/known-issues.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project : Compilation failure:
+
+cannot access org.mule.sdk.api.runtime.streaming.StreamingHelper
+[ERROR] class file for org.mule.sdk.api.runtime.streaming.StreamingHelper not found
+
+cannot find symbol
+[ERROR]   symbol:   method resolveCursors(java.util.HashMap<java.lang.String,java.lang.Object>,boolean)
+[ERROR]   location: variable streamingHelper of type org.mule.runtime.extension.api.runtime.streaming.StreamingHelper
+----
+To solve this issue you must add the sdk.api dependency to the pom.xml:
```
Comment (jmbarbosa80) [inline modules/ROOT/pages/known-issues.adoc]:
```suggestion
To solve this issue you must add the mule-sdk-api dependency to the pom.xml:
```

**Review by dabreu — DISMISSED** [review]

File: modules/ROOT/pages/known-issues.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+To solve this issue you must add the mule-sdk-api dependency to the pom.xml:
+[source,xml,linenums]
+----
+<dependency>
+    <groupId>org.mule.sdk</groupId>
+    <artifactId>mule-sdk-api</artifactId>
+    <version>0.4.0</version>
+</dependency>
+----
+CAUTION: Unless you need new functionality from the new org.mule.sdk.api package, remember to keep using the annotations and classes from *org.mule.runtime.extension.api*. However if you do need any of the new annotations from the org.mule.sdk.api package, you must only use those classes that are annotated with `@MinMuleVersion("4.4")`.
```
Comment (jmbarbosa80) [inline modules/ROOT/pages/known-issues.adoc]:
```suggestion
CAUTION: Unless you need new functionality from the new org.mule.sdk.api package, remember to keep using the annotations and classes from *org.mule.runtime.extension.api*. However, if you do need any of the new annotations from the org.mule.sdk.api package, you must only use those classes that are annotated with `@MinMuleVersion("4.4")`.
```

**Review by jmbarbosa80 — DISMISSED** [review]

File: modules/ROOT/pages/troubleshooting.adoc [hunk 3]
```diff
@@ -3,9 +3,9 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: mule, sdk, troubleshooting, logging
+:keywords: mule, sdk, troubleshooting, logging, known issue, bug, error, limitations
 
-Use logging to troubleshoot SDK components; use verbose logging to test and troubleshoot operations.
+To troubleshoot Java SDK, learn how to enable logging and verbose logging, as well as interpret common error messages.
```
Comment (kbostrom) [inline modules/ROOT/pages/troubleshooting.adoc]:
```suggestion
To troubleshoot Java SDK, learn how to enable logging (including verbose logging) and how to interpret common error messages.
```

**Review by kbostrom — DISMISSED** [review]
CE

**Review by kbostrom — APPROVED** [review]
RA CE

**Pull request merged**